Week1/homework/src/select-query-function.js

```diff
@@ -0,0 +1,29 @@
+'use-strict';
+const util = require('util');
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function select_function(select_query, consoleMessage, column) {
```

Comment:
What if did you change the inputs to column, table, whereCondition, and consoleMessage?

Reply:
Thank you Lukas for your feedback. I appreciate it a lot.
For the inputs , is not clear for me. Do I have to add table input ? and the where condition you mean the query? It would be nice if you can clarify it more for me. 
Thanks,